static/app/views/replays/list/replaySearchBar.tsx

```diff
@@ -91,7 +91,7 @@ function ReplaySearchBar(props: Props) {
         orgSlug: organization.slug,
         tagKey: tag.key,
         search: searchQuery,
-        projectIds: projectIdStrings,
+        projectIds: projectIds.map(String),
```

Comment:
Noticed the old code uses `?`, what happens if projectIds is null here?
Same applies for feedbackSearch

Reply:
I removed them because projectIds is always `number[]` and I believe defaults to an empty array. I've added them back in case.